spring-examples/spring-session/simple-spring-session-client-server/src/main/java/dev/gemfire/spring/session/client/ClientConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015-present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.gemfire.spring.session.client;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.gemfire.config.annotation.ClientCacheApplication;
+import org.springframework.data.gemfire.config.annotation.ClientCacheConfigurer;
+import org.springframework.data.gemfire.support.ConnectionEndpoint;
+import org.springframework.session.data.gemfire.config.annotation.web.http.EnableGemFireHttpSession;
+
+@SpringBootApplication
+@ClientCacheApplication(name = "SpringSessionDataGeodeJavaConfigSampleClient",
```

Comment:
why are you enabling subscriptions if you're not using CQs? Does session require it? Otherwise, you shouldn't need to specify `ClientCacheApplication` at all.

Reply:
In reality it would need it, so that clients can be notified if session information is discarded... But, given that it using a PROXY region, it would not be a huge requirement to add it in.

Reply:
If there is a reason, I don't think it's a problem.